components/lib/confirmdialog/ConfirmDialog.js

```diff
@@ -154,18 +154,21 @@ export const ConfirmDialog = React.memo(
                 getPropValue('rejectClassName')
             );
 
-            const rejectButtonProps = {
-                label: rejectLabel,
-                autoFocus: defaultFocus === 'reject',
-                icon: getPropValue('rejectIcon'),
-                className: classNames(getPropValue('rejectClassName'), cx('rejectButton', { getPropValue })),
-                onClick: reject,
-                pt: ptm('rejectButton'),
-                unstyled: props.unstyled,
-                __parentMetadata: {
-                    parent: metaData
-                }
-            };
+            const rejectButtonProps = mergeProps(
+                {
+                    label: rejectLabel,
+                    autoFocus: defaultFocus === 'reject',
+                    icon: getPropValue('rejectIcon'),
+                    className: classNames(getPropValue('rejectClassName'), cx('rejectButton', { getPropValue })),
+                    onClick: reject,
+                    pt: ptm('rejectButton'),
+                    unstyled: props.unstyled,
+                    __parentMetadata: {
+                        parent: metaData
+                    }
+                },
+                ptm('rejectButton')
+            );
 
```

Comment:
Do you need to do the same for acceptButton?

Reply:
No I don't, because acceptButton already have mergeProps with ptm('acceptButton') and works fine.